packages/babel-cli/src/babel/dir.js

```diff
@@ -71,7 +71,11 @@ module.exports = function (commander, filenames) {
     _.each(filenames, function (dirname) {
       let watcher = chokidar.watch(dirname, {
         persistent: true,
-        ignoreInitial: true
+        ignoreInitial: true,
+        awaitWriteFinish: {
+          stabilityThreshold: 50,
+          pollInterval: 10,
```

Comment:
200 is too close to perception threshold, IMO.

This is not continual poll, and is only active when change event occurs. Performance wise is probably not a big deal.

Works well for me, YMMV.